README.md

```diff
@@ -1,3 +1,5 @@
+# Readme
```

Comment:
do we really want this?

Reply:
I put it in because it makes the doxygen table of contents make more sense. Open to other ideas though.